Add HealthCheckInfo (#366)
submariner-io/submariner-website: src/content/operations/deployment/subctl/_index.en.md

| `--version` `<string>`                  | Image version
<!-- markdownlint-enable line-length -->

> #### `join` flags (HealthCheck)

Reviewer: ```suggestion
#### `join` flags (health check)
```

<!-- markdownlint-disable line-length -->
| Flag                                            | Description
|:------------------------------------------------|:----------------------------------------------------------------------------|

> | `--health-check-enable`                         | Enable Gateway health check (default true)

Reviewer: ```suggestion
| `--health-check-enable`                         | Enable/disable Gateway health check (default true)
```

> | `--health-check-interval` `<uint>`              | The interval in seconds in which health check packets will be send (default 1)

Reviewer: ```suggestion
| `--health-check-interval` `<uint>`              | The interval in seconds at which health check packets will be sent (default 1)
```
